tests/unit/TestPackageJson.js

```diff
@@ -12,7 +12,8 @@ const packageJson = JSON.parse(
 
 // postcss-loader: https://github.com/mozilla/addons-frontend/issues/4592
 // prettier: https://github.com/mozilla/addons-frontend/issues/5312
```

Comment:
Since the new comment now covers both `prettier` and `prettier-quick`, you can delete this one.